qcodes/tests/dataset/test_datasaver.py

```diff
@@ -91,3 +91,16 @@ def test_numpy_types(experiment):
 
     for dtype in dtypes:
         data_saver.add_result(("p", dtype(2)))
+
+
+def test_string(experiment):
+    """
+    Test that we can save text in the data set
+    """
+    p = ParamSpec("p", "text")
+
+    test_set = qc.new_data_set("test-dataset")
+    data_saver = DataSaver(
+        dataset=test_set, write_period=0, parameters={"p": p})
+
+    data_saver.add_result(("p", "some text"))
```

Comment:
don't want to be annoying but:
* pep8 required newline at the end of the file
* should the test also assert that `some text` is indeed in the database?

Reply:
I am quite surprised this commit even passed the quality inspection: Doesn't CI check for PEP8? 

Anyway, I will correct the above.